packages/good-design/src/apps/goodid/controllers/SegmentationController.tsx

```diff
@@ -2,27 +2,53 @@ import React, { useCallback } from "react";
 import {
   AsyncStorage,
   useAggregatedCertificates,
+  useCheckAvailableOffers,
   useFVLink,
   useIdentityExpiryDate,
   useIsAddressVerified,
   useIssueCertificates,
-  useGetEnvChainId
+  useGetEnvChainId,
+  useCertificatesSubject
 } from "@gooddollar/web3sdk-v2";
 import { useEthers } from "@usedapp/core";
 import { isEmpty } from "lodash";
 
-import { SegmentationWizard } from "../wizards/SegmentationWizard";
-import { useCertificatesMap } from "../../../hooks";
+import { SegmentationProps, SegmentationWizard } from "../wizards/SegmentationWizard";
 
-export const SegmentationController = () => {
+const redtentOffer = [
+  {
+    campaign: "RedTent",
```

Comment:
what about age?

Reply:
If I understood the call on tuesday correctly, age was no longer a criteria to match again.

@decentralauren can you re-confirm?